apps/web/src/store/modules/file/actions.ts

```diff
@@ -116,3 +116,15 @@ export const downloadExcel: Action<FileState, any> = async ({ commit, rootState,
         dispatch('display/finishLoading', {}, { root: true });
     }
 };
+
+export const downloadByFileUrl: Action<ExportResponse, any> = async (_, downloadUrl): Promise<void> => {
+    try {
+        const link = document.createElement('a');
+        link.href = downloadUrl;
+        link.target = '_self';
+        link.click();
+        link.remove();
+    } catch (error) {
+        ErrorHandler.handleRequestError(error, i18n.t('COMMON.EXCEL.ALT_E_DOWNLOAD'));
```

Comment:
suggestion: The name of the function is 'downloadByFileUrl' but the error message is about excel feature(COMMON.EXCEL.ALT_E_DOWNLOAD).

I think it would be far better if it is changed either in two ways:
1. Change the function name that includes 'excel' keyword to use it only for excel export case.
2. Accept an error message as an argument or other argument that indicates the file format.